src/lib/wasm/ConstellationStore.ts

```diff
@@ -91,6 +91,25 @@ class ConstellationStore {
         return failure(WarpError.CONSTELLATION_NOT_FOUND)
     }
 
+    /**
+     * moves item/s from constellation.
+     * @returns A Result containing either success or failure with a WarpError.
+     */
+    async dropIntoFolder(fileName: string, toFolderName: string): Promise<Result<WarpError, void>> {
+        const constellation = get(this.constellationWritable)
+        let currentDir = constellation!.current_directory()
+        if (constellation) {
+            try {
+                currentDir.move_item_to(fileName, toFolderName)
```

Comment:
Since we are checking `constellation` we should move this within the if statement. Furthermore, since `Constellation::current_directory` could yield an error, we should move it within the try/catch scope to catch any possible errors. 
```suggestion
        if (constellation) {
            try {
                let currentDir = constellation!.current_directory()
                currentDir.move_item_to(fileName, toFolderName)
```

Reply:
Doh! Thank you Darius! I missed changing that back when I was furiously trying everything I could think of for the move